Loop/Models/RemoteCommand.swift

```diff
@@ -8,15 +8,42 @@
 
 import Foundation
 import LoopKit
+import HealthKit
 
```

Comment:
I don't believe this was being used from what I can tell. But it looks like the right structure to track remote command errors like I needed.